apps/electron/src/menu/applicationMenu.js

```diff
@@ -150,6 +150,12 @@ function getApplicationMenu(isMac, askToQuit, urlBase) {
     {
       role: 'help',
       submenu: [
+        {
+          label: 'About',
+          click: async () => {
+            redirectCb('editor?settings=about');
+          },
+        },
```

Comment:
could we show the app version here somewhere? we would need to retrieve it from the package.json in a similar way we do in the client / server